flow/project.py

```diff
@@ -892,7 +902,7 @@ def evaluate(value):
     def _submit_cmd(self, entrypoint, ignore_conditions, jobs=None):
         entrypoint = self._determine_entrypoint(entrypoint, dict(), jobs)
         cmd = "{} run -o {}".format(entrypoint, self.name)
-        cmd = cmd if jobs is None else cmd + ' -j {}'.format(' '.join(map(str, jobs)))
+        cmd = cmd if jobs is None else cmd + f' -j {get_aggregate_id(jobs)}'
```

Comment:
This might be clear to other people but it hurt my brain a bit/it was hard for me to see this is an f-string, especially since the line above uses the `.format()` syntax.

Reply:
The github syntax highlighting might be the real issue, but I wonder if it would be more clear to use an f-string one line 904 as well, since its the only example in this function of using the `.format()` syntax